mangaki/mangaki/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class MangakiConfig(AppConfig):
+    name = 'mangaki'
+    verbose_name = 'Mangaki'
+
+    def ready(self):
+        from . import receivers
```

Comment:
It ensures that the `mangaki.receivers` module is loaded, which ensures that the signal receivers decorated with `@receiver` are connected when the app is ready.